SabreTools.DatFiles/Formats/SoftwareList.cs

```diff
@@ -332,8 +337,10 @@ private bool ReadPart(XmlReader reader, Machine machine, Part part, bool statsOn
                         List<DatItem> roms = ReadDataArea(reader.ReadSubtree(), dataArea);
 
                         // If we got valid roms, add them to the list
-                        if (roms != null)
-                            items.AddRange(roms);
+                        if (!String.IsNullOrEmpty(string.Join(", ", roms)))
```

Comment:
I'd prefer consistency here in using `String` and `string` with a preference being on `string`.